test/utest.cpp

```diff
@@ -337,6 +362,56 @@ TEST(MultiClassLoaderTest, noWarningOnLazyLoad) {
   SUCCEED();
 }
 
+// This is a different class name so that static variables in ClassLoader are isolated
+TEST(ClassLoaderGraveyardTest, loadUnloadLoadFromGraveyard) {
+#ifdef _WIN32
+  // Not run on Windows because this tests dlopen-specific behavior
+  GTEST_SKIP();
+#endif
+
+  // This first load/unload adds the plugin to the graveyard
+  try {
+    class_loader::ClassLoader loader(GLOBAL_PLUGINS, false);
+    loader.createInstance<Base>("Kangaroo")->saySomething();
+    loader.unloadLibrary();
+  } catch (class_loader::ClassLoaderException & e) {
+    FAIL() << "ClassLoaderException: " << e.what() << "\n";
+  }
+
+  // Not all platforms use RTLD_GLOBAL as a default, and rcutils doesn't explicitly choose either.
+  // In order to invoke graveyard behavior, this needs to be loaded first for global relocation.
+#ifndef _WIN32
```

Comment:
you are already skipping the test when the platform is windows, why do you need this check and other the one below ?

Reply:
It won't compile otherwise because dlopen/dlclose don't exist on windows.

Reply:
Why not else putting the entire test in an `#else` clause after `#ifdef`?

Reply:
I suppose it's simpler inside a single ifndef block. Fixed.